Makefile

```diff
@@ -0,0 +1,48 @@
+.PHONY: clean clean-test clean-pyc clean-build docs help
+.DEFAULT_GOAL := help
+
+define BROWSER_PYSCRIPT
+import os, webbrowser, sys
+
+from urllib.request import pathname2url
+
+webbrowser.open("file://" + pathname2url(os.path.abspath(sys.argv[1])))
+endef
+export BROWSER_PYSCRIPT
+
+define PRINT_HELP_PYSCRIPT
+import re, sys
+
+for line in sys.stdin:
+	match = re.match(r'^([a-zA-Z_-]+):.*?## (.*)$$', line)
+	if match:
+		target, help = match.groups()
+		print("%-20s %s" % (target, help))
+endef
+export PRINT_HELP_PYSCRIPT
+
+BROWSER := python -c "$$BROWSER_PYSCRIPT"
+
+help:
+	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
+
+clean: clean-build clean-pyc clean-test ## remove all build, test, coverage and Python artifacts
+
+autodocs: ## generate Sphinx HTML documentation, including API docs
+	rm -f doc/spyder.*
+	rm -f doc/modules.rst
+	sphinx-apidoc -o doc/ spyder-repo/spyder/ *tests*
+
+docs: ## generate Sphinx HTML documentation, including API docs
+	$(MAKE) -C doc clean
+	$(MAKE) -C doc html
+	# $(MAKE) -C doc linkcheck
```

Comment:
Why was this disabled? Without it, links are likely to go stale and die.

Reply:
No idea, I re-added it

Reply:
Possibly for local testing since there were some broken links I've since fixed. Thanks!